platform/error/platform-error-impl/src/main/resources/codeMessages.properties

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Codice Foundation
+#
+# This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+# version 3 of the License, or any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+# <http://www.gnu.org/licenses/lgpl.html>.
+
+403=You are not authorized to see the resource you are attempting to view. Please \
+  <a href="../login/index.html?prevurl={{uri}}">Sign In</a> to continue.
+404=The page you are requesting could not be found on this server. Ensure that the URL was typed \
+  in correctly and try again.
+405=The method specified in the Request-Line is not allowed for the resource identified by the \
+  Request-URI. The response MUST include an Allow header containing a list of valid methods for \
+  the requested resource.
+406=The resource identified by the request is only capable of generating response entities which \
+  have content characteristics not acceptable according to the accept headers sent in the request.
+401=You are not authenticated. Please <a href="../login/index.html?prevurl={{uri}}">Sign In</a> to \
+  continue.
+500=The server encountered an unexpected condition which prevented it from fulfilling the request.
+501=The server does not support the functionality required to fulfill the request.
+502=The server, while acting as a gateway or proxy, received an invalid response from the upstream \
+  server it accessed in attempting to fulfill the request.
+503=The server is currently unable to handle the request due to a temporary overloading or \
+  maintenance of the server.
```

Comment:
✏️ It seems that this list of status codes is missing `400` and `504`, which we have handled in the past. I'm not sure about the value of `504`, but I see a lot of value in `400`.